src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ConstructedEETypeNode.cs

```diff
@@ -168,6 +166,8 @@ protected override DependencyList ComputeNonRelocationBasedDependencies(NodeFact
             // Ask the metadata manager if we have any dependencies due to reflectability.
             factory.MetadataManager.GetDependenciesDueToReflectability(ref dependencyList, factory, _type);
 
+            CodeBasedDependencyAlgorithm.AddInterestingInteropConstructedTypeDependencies(ref dependencyList, factory, _type);
```

Comment:
This should call into the interop manager, same as we do for metadata dependencies on the line above.